CONTRIBUTING.md

```diff
@@ -153,10 +154,25 @@ It is important to note that, for authorship, these should be substantial intell
 
 ## Peer review
 
-All pull requests will undergo peer review.
-Participants in this project should review pull requests, which can be done using [GitHub's review interface](https://help.github.com/articles/about-pull-request-reviews/ "GitHub: about pull request reviews").
-They should suggest modifications or, potentially, directly edit the pull request to make suggested changes.
-As a reviewer, it's helpful to note the type of review you performed: did you look over the source code, did you run the source code, did you look at and interpret the results or a combination of these?
+All pull requests will undergo peer review. The review intends to ensure that a pull request conforms to the following basic requirements.
+
+- The code is correct.
+- The results can be reproduced in a reasonable amount of time.
+- The results are expected.
+- The documentation describes the purpose, methods, results, input, output, and how to run the code.
+- The code follows basic style guidelines.
+- The code contains comments that explain non-obvious procedures.
+
+Participants in this project should review pull requests, which can be done using [GitHub's review interface](https://help.github.com/articles/about-pull-request-reviews/ "GitHub: about pull request reviews"). Following are the basic guidelines for reviewing pull requests:
+
+- Note the type of review you performed: did you look over the source code, did you look over the documentation, did you run the source code, did you look at and interpret the results or a combination of these?
+- Suggest modifications or, potentially, directly suggest, _but do not commit changes to_, the pull request, so that the pull request conforms to the aforementioned basic requirements.
+- Explain in **detail** why the suggested modifications are **necessary**, and how the modifications should be implemented specifically, so that the developers are able to follow the suggestions without misunderstanding.
```

Comment:
Oh I get it now - how about the following:
```suggestion
- Explain in **detail** why the suggested modifications are **necessary**, and how the modifications should be implemented specifically, so that the developers are able to follow the suggestions without misunderstanding.
- For any suggestions/comments that are related to the lines that were changed in the particular PR, make the comment underneath the code by clicking the "+" next to the code. Click "Start a review" for the first comment, and afterwards, click "Add single comment" to keep all comments within the same review. Click "Finish your review" once completed. 
- If there is any comments/suggestions that are **outside** of the code changes this particular PR, for example, any questions about the results, or any lines of code (although not changed) that seemed puzzling, mentioned them as comments after you click "Finish your review". 
```

Reply:
Thank you for the clear descriptions @runjin326 ! Sorry that I thought what you described at <https://github.com/PediatricOpenTargets/OpenPedCan-analysis/pull/73#discussion_r681241142> was in a more specific context.

I think maybe we could also use permalinks for this purpose, which is included in the following suggestion. I was wondering if it makes sense.

```suggestion
- Explain in **detail** why the suggested modifications are **necessary**, and how the modifications should be implemented specifically, so that the developers are able to follow the suggestions without misunderstanding.
- For any suggestions/comments that are related to the lines that were changed in the particular PR, make the comment underneath the code by clicking the "+" next to the code. Click "Start a review" for the first comment, and afterwards, click "Add single comment" to keep all comments within the same review. Click "Finish your review" once completed. 
- If there is any comments/suggestions that are **outside** of the code changes this particular PR, for example, any questions about the results, or any lines of code (although not changed) that seemed puzzling, mentioned them as comments after you click "Finish your review". The permalinks of the referenced results or code can be obtained by the following procedure: click the "Commits" tab of the pull request, browse the repository at a specific commit that contains the referenced results or code by clicking the `<>` button of the commit row, go to the referenced file in the repository to get file or line permalinks.
```

Reply:
Hey Yuanchao @logstar, yes using the permalink would be a great idea!